app/scripts/modules/core/src/pipeline/config/templates/configurePipelineTemplateModal.controller.ts

```diff
@@ -119,6 +119,10 @@ export class ConfigurePipelineTemplateModalController implements IController {
       .map(v => {
         if (v.type === 'object') {
           v.value = load(v.value);
+        } else if (v.type === 'int') {
+          return [v.name, parseInt(v.value)];
```

Comment:
The base should be explicit here `parseInt(..., 10)`, since apparently the default is browser-specific.

Reply:
TIL thanks!